app/pages/index.tsx

```diff
@@ -152,6 +152,9 @@ export default class Index extends Component<Props> {
           </Col>
         </div>
         <style jsx>{`
+          .row {
+            padding-top: 50px;
+          }
```

Comment:
We only want to add the padding to the very first row (`:first-child`).

I did also forget to mention that we don't need or want the extra padding on smaller screens, which is why I put the suggestion above in a media query like so:
```
@media screen and (min-width: 992px) {
  .row:first-child {
    padding-top: 50px;
  }
}
```

Reply:
Right, sorry my coffee hasn't kicked in yet :sleepy: